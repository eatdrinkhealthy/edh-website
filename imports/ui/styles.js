import { css } from "glamor";

//
// Component Style Pattern (use with glamor)
//
// const componentRules = {
//    rule1: "value",
//      ...
//    ruleN: "value",
// };
// export const componentStyle = {
//    rules: componentRules,
//    className: `${css(componentRules)}`,
// };
//

const appLinksHeaderRules = {
  marginTop: "90px",
  marginLeft: "40px",
  fontSize: "16px",
  color: "darkbrown",
};
export const appLinksHeaderStyle = {
  rules: appLinksHeaderRules,
  className: `${css(appLinksHeaderRules)}`,
};

const appLinksButtonsRules = {
  marginTop: "14px",
  marginBottom: "20px",
  marginLeft: "40px",
};
export const appLinksButtonsStyle = {
  rules: appLinksButtonsRules,
  className: `${css(appLinksButtonsRules)}`,
};

const linkButtonRules = {
  backgroundColor: "darkslategray",
  border: "none",
  borderRadius: "4px",
  color: "white",
  padding: "8px 32px 8px 32px",
  marginRight: "8px",
  textAlign: "center",
  textDecoration: "none",
  fontSize: "12px",
};
export const linkButtonStyle = {
  rules: linkButtonRules,
  className: `${css(linkButtonRules)}`,
};
