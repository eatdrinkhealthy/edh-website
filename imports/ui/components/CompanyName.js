import React from "react";
import { companyNameStyle } from "../styles";

const CompanyName = () => (
  <div className={companyNameStyle.css} />
);

CompanyName.style = companyNameStyle;

export default CompanyName;
