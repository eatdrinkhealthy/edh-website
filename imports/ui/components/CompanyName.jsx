import React from "react";
import { companyNameStyle } from "../styles";

const CompanyName = () => (
  <div className={companyNameStyle.className} />
);

export default CompanyName;
