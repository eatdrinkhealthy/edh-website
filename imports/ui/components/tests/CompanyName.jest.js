/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */

import React from "react";
import renderer from "react-test-renderer";
import CompanyName from "../CompanyName";
import { CompanyNameStyle } from "../../styles";

describe("<CompanyName />", function () {
  it("render matches snapshot", function () {
    const tree = renderer.create(<CompanyName />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("css matches snapshot", function () {
    expect(JSON.stringify(CompanyNameStyle, null, 2)).toMatchSnapshot();
  });
});
