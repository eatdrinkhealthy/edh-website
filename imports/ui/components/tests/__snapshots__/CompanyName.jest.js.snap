exports[`<CompanyName /> css matches snapshot 1`] = `undefined`;

exports[`<CompanyName /> render matches snapshot 1`] = `
<div
  className={
    Object {
      "data-css-nkpr7m": "",
    }
  } />
`;
